google-cloud-firestore/src/main/java/com/google/cloud/firestore/DocumentSnapshot.java

```diff
@@ -448,4 +448,11 @@ public boolean equals(Object obj) {
   public int hashCode() {
     return Objects.hash(rpcContext, docRef, fields);
   }
+
+  @Override
+  public String toString() {
+    return String.format(
+        "%s{docRef=%s, fields=%s, readTime=%s, updateTime=%s, createTime=%s}",
```

Comment:
Here `docRef` is used, and in `UpdateBuilder.java` `documentReference` is used. How about just use `doc` consistently?

Reply:
These are just the field names, which happen to be different.

Reply:
Right, but that implementation detail doesn't need to bleed out through the string returned from `toString()`. Since callers of `toString()` don't necessarily know, or even need to know, what the internal private variable name is, it's best to be consistent from that perspective.